src/bindings/python/flux/job.py

```diff
@@ -267,6 +267,64 @@ def wait(flux_handle, jobid=lib.FLUX_JOBID_ANY):
     return future.get_status()
 
 
+def kill_async(flux_handle, jobid, signum=None):
+    """Send a signal to a running job asynchronously
+
+    :param flux_handle: handle for Flux broker from flux.Flux()
+    :type flux_handle: Flux
+    :param jobid: the job ID of the job to kill
+    :param signum: signal to send (default SIGTERM)
+    :returns: a Future
+    :rtype: Future
+    """
+    if not signum:
+        import signal
+
+        signum = signal.SIGTERM
+    return Future(RAW.kill(flux_handle, int(jobid), signum))
+
+
+def kill(flux_handle, jobid, signum=None):
+    """Send a signal to a running job.
+
+    :param flux_handle: handle for Flux broker from flux.Flux()
+    :type flux_handle: Flux
+    :param jobid: the job ID of the job to kill
+    :param signum: signal to send (default SIGTERM)
+    :returns: a Future
+    :rtype: Future
+    """
+    return kill_async(flux_handle, jobid, signum).get()
+
+
+def cancel_async(flux_handle, jobid, reason=None):
+    """Cancel a pending or or running job asynchronously
+
+    :param flux_handle: handle for Flux broker from flux.Flux()
+    :type flux_handle: Flux
+    :param jobid: the job ID of the job to cancel
+    :returns: a Future
+    :rtype: Future
+
+    """
+    if not reason:
+        reason = ffi.NULL
+    return Future(RAW.cancel(flux_handle, int(jobid), reason))
+
+
+def cancel(flux_handle, jobid, signum=None):
+    """Cancel a pending or or running job
+
+    :param flux_handle: handle for Flux broker from flux.Flux()
+    :type flux_handle: Flux
+    :param jobid: the job ID of the job to cancel
+    :returns: a Future
+    :rtype: Future
```

Comment:
I presume this was just a copy-paste carryover and that the `.get()` doesn't return anything?

Reply:
yes, good catch.